test/test_noipy.py

```diff
@@ -31,14 +31,12 @@ def tearDown(self):
             shutil.rmtree(self.test_dir)
 
     def test_get_ip(self):
-        ip = utils.get_ip()
+        ip = utils.get_ip('https://httpbin.org/ip')
 
         self.assertTrue(re.match(VALID_IP_REGEX, ip), 'get_ip() failed.')
 
         # monkey patch for testing (forcing ConnectionError)
-        utils.HTTPBIN_URL = 'http://example.nothing'
-
-        ip = utils.get_ip()
+        ip = utils.get_ip('http://example.nothing')
```

Comment:
Please, take a look at the [above comment](https://github.com/pv8/noipy/pull/203/files#diff-3cbc2bd3e1855fb591c4eba9db82ae274a8670d4b961cf02f80a0818f54ce884R22).

```suggestion
        ip = utils.get_ip(req_ip_url='http://example.nothing')
```